electrum/wallet.py

```diff
@@ -1845,6 +1845,17 @@ def get_new_sweep_address_for_channel(self) -> str:
         assert is_address(selected_addr), f"not valid bitcoin address: {selected_addr}"
         return selected_addr
 
+    def should_keep_reserve_utxo(self, to_be_sent: int, is_channel_opening: bool) -> bool:
```

Comment:
we should try to suffix amounts in function params with the unit:
```suggestion
    def should_keep_reserve_utxo(self, to_be_sent_sat: int, is_channel_opening: bool) -> bool:
```